<h1>Taxonomy classifiers for use with q2-feature-classifier</h1>

<div class="danger panel">
    <h4>Danger</h4>
    <p>
        Pre-trained classifiers that can be used with <span class="code">q2-feature-classifier</span> currently
        present a security risk. If using a pre-trained classifier such as the ones
        provided here, you should trust the person who trained the classifier and
        the person who provided you with the qza file. This security risk will be
        addressed in a future version of <span class="code">q2-feature-classifier</span>.
    </p>
</div>

<div class="note panel">
    <h4>Note</h4>
    <p>
        Taxonomic classifiers perform best when they are trained based on your
        specific sample preparation and sequencing parameters, including the primers
        that were used for amplification and the length of your sequence reads.
        Therefore in general you should follow the instructions in
        <a class="note_a" href="https://docs.qiime2.org/2024.2/tutorials/feature-classifier/">Training feature classifiers with q2-feature-classifier</a>
        to train your own taxonomic classifiers
        (for example, from the marker gene reference databases below).
    </p>
</div>

<h2>Naive Bayes classifiers trained on:</h2>
<ul>
    <li><a href="https://data.qiime2.org/2024.2/common/silva-138-99-nb-classifier.qza">Silva 138 99% OTUs full-length sequences</a> (MD5: b8609f23e9b17bd4a1321a8971303310)</li>
    <li><a href="https://data.qiime2.org/2024.2/common/silva-138-99-515-806-nb-classifier.qza">Silva 138 99% OTUs from 515F/806R region of sequences</a> (MD5: e05afad0fe87542704be96ff483824d4)</li>
    <li><a href="https://data.qiime2.org/classifiers/greengenes/gg_2022_10_backbone_full_length.nb.qza">Greengenes2 2022.10 full length sequences</a> (MD5: 98d34227fe67b34f62b464466cca4ffa)</li>
    <li><a href="https://data.qiime2.org/classifiers/greengenes/gg_2022_10_backbone.v4.nb.qza">Greengenes2 2022.10 from 515F/806R region of sequences</a> (MD5: 43de361005ae6dcae61b078c0c835021)</li>
</ul>

<div class="note panel">
    <h4>Note</h4>
    <p>
        Greengenes2 has succeeded Greengenes 13_8. If you still need to access
        the outdated 13_8 classifiers, for example to reproduce old results or
        to compare against new classifiers, you can access them through the older
        QIIME 2 data resources pages.
    </p>
</div>

<div class="note panel">
    <h4>Note</h4>
    <p>
        The Silva classifiers provided here include species-level taxonomy. While
        Silva annotations do include species, Silva does not curate the species-level
        taxonomy so this information may be unreliable. In a future version of
        QIIME 2 we will no longer include species-level information in our Silva
        taxonomy classifiers. This is discussed on the QIIME 2 Forum
        <a class="note_a" href="https://forum.qiime2.org/t/processing-filtering-and-evaluating-the-silva-database-and-other-reference-sequence-data-with-rescript/15494#heading--second-header">here</a>
        (see Species-labels: caveat emptor!).
    </p>
</div>

<p>For Silva 138, please cite the following references if you use any of these pre-trained classifiers:</p>
<ul>
    <li>Michael S Robeson II, Devon R O’Rourke, Benjamin D Kaehler, Michal Ziemski, Matthew R Dillon, Jeffrey T Foster, Nicholas A Bokulich. RESCRIPt: Reproducible sequence taxonomy reference database management for the masses. bioRxiv 2020.10.05.326504; doi: <a href="https://doi.org/10.1101/2020.10.05.326504">https://doi.org/10.1101/2020.10.05.326504</a></li>
    <li>Bokulich, N.A., Kaehler, B.D., Rideout, J.R. et al. Optimizing taxonomic classification of marker-gene amplicon sequences with QIIME 2’s q2-feature-classifier plugin. Microbiome 6, 90 (2018). <a href="https://doi.org/10.1186/s40168-018-0470-z">https://doi.org/10.1186/s40168-018-0470-z</a></li>
    <li>>See the <a href="https://www.arb-silva.de/">SILVA website for the latest citation information for this reference database.</a></li>
</ul>
<p>For Greengenes2, please cite:<p>
<ul>
    <li>McDonald, D. et al. Greengenes2 unifies microbial data in a single reference tree. Nature Biotechnology (2023). <a href="https://www.nature.com/articles/s41587-023-01845-1">https://www.nature.com/articles/s41587-023-01845-1</a></li>
</ul>
<p>If using the Naive Bayes classifiers with Greengenes2, please cite:</p>
<ul>
    <li>Bokulich, N.A., Kaehler, B.D., Rideout, J.R. et al. Optimizing taxonomic classification of marker-gene amplicon sequences with QIIME 2’s q2-feature-classifier plugin. Microbiome 6, 90 (2018). <a href="https://www.nature.com/articles/s41587-023-01845-1">https://doi.org/10.1186/s40168-018-0470-z</a></li>
</ul>
<p>
    Please note, these classifiers were trained using scikit-learn 0.24.1, and
    therefore can only be used with scikit-learn 0.24.1. If you observe errors
    related to scikit-learn version mismatches, please ensure you are using the
    pretrained-classifiers that were published with the release of QIIME 2 you
    are using.
</p>

<h2>Weighted Taxonomic Classifiers</h2>
<p>
    These 16S rRNA gene classifiers were trained with weights that take into
    account the fact that not all species are equally likely to be observed.
    If your sample comes from any of the 14 habitat types we tested, these
    weighted classifiers should give you superior classification precision.
    If your sample doesn't come from one of those habitats, they might still
    help. If you have the time, training with weights specific to your habitat
    should help even more. Weights for a range of habitats
    <a href="https://github.com/BenKaehler/readytowear">are available here</a>
</p>
<ul>
    <li><a href="https://data.qiime2.org/2024.2/common/silva-138-99-nb-weighted-classifier.qza">Weighted Silva 138 99% OTUs full-length sequences</a> (MD5: 48965bb0a9e63c411452a460d92cfc04)</li>
    <li><a href="https://data.qiime2.org/2024.2/common/gg-13-8-99-nb-weighted-classifier.qza">Weighted Greengenes 13_8 99% OTUs full-length sequences</a> (MD5: 2baf87fce174c5f6c22a4c4086b1f1fe)</li>
    <li><a href="https://data.qiime2.org/2024.2/common/gg-13-8-99-515-806-nb-weighted-classifier.qza">Weighted Greengenes 13_8 99% OTUs from 515F/806R region of sequences</a> (MD5: 8fb808c4af1c7526a2bdfaafa764e21f)</li>
</ul>

<div class="note panel">
    <h4>Note</h4>
    <p>
        The Silva classifiers provided here include species-level taxonomy. While
        Silva annotations do include species, Silva does not curate the species-level
        taxonomy so this information may be unreliable. In a future version of
        QIIME 2 we will no longer include species-level information in our Silva
        taxonomy classifiers. This is discussed on the QIIME 2 Forum
        <a class="note_a" href="https://forum.qiime2.org/t/processing-filtering-and-evaluating-the-silva-database-and-other-reference-sequence-data-with-rescript/15494#heading--second-header">here</a>
        (see Species-labels: caveat emptor!).
    </p>
</div>

<p>Please cite the following reference, in addition to those listed above, if you use any of these weighted pre-trained classifiers:</p>
<ul>
    <li>, B.D., Bokulich, N.A., McDonald, D. et al. Species abundance information improves sequence taxonomy classification accuracy. Nature Communications 10, 4643 (2019). <a href="https://doi.org/10.1038/s41467-019-12669-6">https://doi.org/10.1038/s41467-019-12669-6</a></li>
</ul>

<style>
    .code {
        background-color: #F5F5F5;
        font-family: monospace;
        padding-top: 2px;
        padding-right: 4px;
        padding-bottom: 2px;
        padding-left: 4px;
    }

    .panel {
        padding: 7px;
        margin-top: 10px;
        margin-bottom: 10px;
    }

    .danger {
        color: #A94442;
        background-color: #F2DeDe;
        border: 1px solid #EBCCCC;
    }

    .note {
        color: #3A87AD;
        background-color: #D9EDF7;
        border: 1px solid #BCE8F1;
    }

    .note_a {
        font-weight: 700;
    }

    a {
        color: #337AB7;
    }

    a:hover {
        color: #23527C;
    }
</style>
